### ISSUE TITLE
Does  it work for mlp ？

### ISSUE THREAD

**Issue by huajianxiaowanzi** [intel/scikit-learn-intelex#1104]
Does  it work for MLPClassifier ？I tested an example and it didn't work

**Comment by huajianxiaowanzi** [comment]
```import numpy as np
from sklearn.neural_network import MLPClassifier
from timeit import default_timer as timer
clf = MLPClassifier(hidden_layer_sizes=(128, 32), activation='relu', solver='adam', alpha=0.01,
                    batch_size=70, learning_rate_init=0.001, max_iter=100, shuffle=True,
                    random_state=42, tol=0.0001, verbose=True, warm_start=False,
                    early_stopping=True, validation_fraction=0.1, beta_1=0.9, beta_2=0.999, epsilon=1e-08)
num_samples = 200000
X = np.random.rand(num_samples, 8)
y = np.random.randint(2, size=(num_samples, 1))

from sklearnex import unpatch_sklearn,patch_sklearn
unpatch_sklearn()
start = timer()
clf.fit(X,y)
train_unpatched = timer() - start
f"Original Scikit-learn time: {train_unpatched:.2f} s"

patch_sklearn()
start = timer()
clf.fit(X,y)
train_patched = timer() - start
f"Intel® extension for Scikit-learn time: {train_patched:.2f} s"

**Comment by Alexsandruss** [comment]
MLP is not supported by scikit-learn-intelex.
See [Supported Algorithms](https://intel.github.io/scikit-learn-intelex/algorithms.html) for list of all supported algorithms.
